### PR TITLE
feat(core): Add stop model sync to admin api

### PR DESCRIPTION
**Pull request by stephhuynh18**

### CODE REVIEW

File: packages/core/src/sync/sync-api.ts [hunk 1]
```diff
@@ -365,6 +365,19 @@ export class SyncApi implements ISyncApi {
     })
   }
 
+  /**
+   * Stop models from being included in the continuous sync
+   * TODO (CDB-2303): Remove existing history sync jobs as well
```
Comment (stbrody) [inline packages/core/src/sync/sync-api.ts:370]: could we log a warning that the sync will keep running to completion, and mention that this is a temporary state, and link a TODO to delete the log message when this is done?

**Review by stbrody — APPROVED** [review]
lgtm.  Seems like this should go into the ComposeDB beta RC we're cutting today, yes?

CC @oed

**Review by willex — APPROVED** [review]
RC📦✅

**Comment by stbrody** [comment]
FYI test is failing @stephhuynh18

**Pull request merged**